reports/RJ722-II.md

```diff
@@ -0,0 +1,439 @@
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/22fbb2281503fc75b8960df528e878e2cfa24ed3
+   - details: Add docs and news
+
+ - 6:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/7fc63f7097ba83093740b4b3c1eea788fd485dd1
+   - details: Package codet
+
+ - 7:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/680cc58051d3640b73f9ae1ce936193843b9b604
+   - details: Ignore dist/
+
+ - 8:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/111bc5d5cab4c516a7eecb75dd637722f1b1aa37
+   - details: Remove nargs=1
+
+ - 9:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/cbdbd4e391925b69c93d64c794d925422b9a10e6
+   - details: Minor Cosmetics
+
+ - 10:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/f8d854059f740d14147dc92906842e5c1bc04c34
+   - details: Supply packages in setup.py
+
+ - 11:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/50e5dc1ec3578d02ac01804fda733bdec612998f
+   - details: Ignore build/
+
+ - 12:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/836b31fd0a21407224c0f7d7381d1d03047b22f8
+   - details: Ignore MANIFEST
+
+ - 13:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/1ec97c5eb83ad7cb0872f96931309fc46e0d8caf
+   - details: Set up .gitignore
+
+ - 14:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/186498c4ff7c91811620a5d6152ac8b2b2253eb7
+   - details: Add setup.cfg
+
+ - 15:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/c6c25145b9bdd8372b4f4388eddbbbeeab13ff7b
+   - details: Rename genereators/qt_utils.py to generators/qt.py
+
+ - 16:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/1fccd02ac6fdd893801df0dc498b96b22ef976af
+   - details: Rename generators to vulture_whitelist
+
+ - 17:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/67c3f185452bfb222ff0e78ae6ddb3b15a04782b
+   - details: Use QtWhitelistCreator as a Creator object
+
+ - 18:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/473df29f75460120bbcd97dbb328ac5277032a71
+   - details: Add tests and configure test suite
+
+ - 19:
+   - repo: vulture-whitelist-generators
+   - link: https://github.com/RJ722/vulture-whitelist-generators/commit/8de282f0d31fb8e46e2eb96ef93b48004e60ab85
```

Comment:
this type of addition is being excluded by other students as it didnt involve any additional work

Reply:
Yups, this should be excluded - removing it.